logger/buffers/impl.go

```diff
@@ -0,0 +1,84 @@
+package buffers
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/ironsmile/nedomi/config"
+)
+
+// New returns a new Buffers logger.
+func New(cfg *config.LoggerSection) (*Buffers, error) {
+	b := &Buffers{
+		buffer: new(bytes.Buffer),
+	}
+	return b, nil
+}
+
+// Buffers writes all the logs to
+// bytes.Buffer for later examination
+type Buffers struct {
+	buffer *bytes.Buffer
```

Comment:
For this logger to be useful as a mock logger, we need some way to get the logged messages externally. Either this has to be exported or we need a getter method. Also, why use a buffer? Why not just a slice of strings that we append to?